images/node/image.yaml

```diff
@@ -3,38 +3,8 @@ terraform: false
 
 versions:
   - apko:
-      extractTagsFrom:
-        package: nodejs-18
-      config: configs/latest.apko.yaml
-      subvariants:
-        - suffix: -dev
-          options:
-            - dev
-            - node-dev
+      config: configs/18.apko.yaml
   - apko:
       config: configs/19.apko.yaml
-      extractTagsFrom:
-        package: nodejs-19
-      subvariants:
-        - suffix: -dev
-          options:
-            - dev
-            - node-dev
   - apko:
       config: configs/20.apko.yaml
```

Comment:
We should kill these because they all do the same thing, and it'll spawn 3 competing legs